bin/pycbc_inspiral

```diff
@@ -25,6 +25,7 @@ from pycbc.types import TimeSeries, FrequencySeries, zeros, float32, complex64
 import pycbc.fft.fftw, pycbc.version
 import pycbc.opt
 import pycbc.weave
+import pycbc.injcutter
```

Comment:
I never said that all the relevant sub-options have to begin with the whole string 'injection-skip-templates', did I.  --injection-chirp-time-threshold might be sufficient.  (BTW, should be called a chirp time _window_ since it's a threshold on the _difference_ in tau0.)

Anyway, I don't think it can be clear unless 'injection' is in the name, because the whole thing can only run on injections.  '(Template) filter rejector' doesn't mean injection.